octobot_evaluators/evaluator/TA_evaluator.py

```diff
@@ -52,7 +47,40 @@ async def start(self, bot_id: str) -> bool:
             self.logger.error("Can't connect to OHLCV trading channel")
         return False
 
+    async def reset_evaluation(self, cryptocurrency, symbol, time_frame):
+        self.eval_note = START_PENDING_EVAL_NOTE
+        await self.evaluation_completed(cryptocurrency, symbol, time_frame, eval_time=0)
```

Comment:
It will push to all the matrix consumer that the evaluator reset its eval node ?

Reply:
We need a way to invalidate the previous eval (for the TA updater cycle not to consider the previous eval valid) before starting to push new evaluations based on the in-construction candles.
I used this for now but there are other ways to do it, we could also tell the matrix to invalidate an evaluation for example. Instead of using a channel for that

Reply:
I think a `should_notify=False` to prevent matrix pushing should do the trick